tests/integration/security/testdata/requestauthn/remote.yaml.tmpl

```diff
@@ -9,7 +9,7 @@ spec:
       app: {{ .dst }}
   jwtRules:
   - issuer: "test-issuer-1@istio.io"
-    jwksUri: "http://jwt-server.{{ .Namespace }}.svc.cluster.local:8000/jwks"
+    jwksUri: "https://jwt-server.default.svc.cluster.local:8443/jwks"
```

Comment:
Can you comment on why replacing {{ .Namespace }} with a hardcoded namespace "default"?
Should a new jwksUri be created for the new httpbs-based test case, instead of revising an existing jwksUri?

Reply:
We haven't specified any ns in the deployment, it will always use default namespace. That's my reasoning, correct me if I'm wrong here.

Reply:
As {{ .Namespace }} will always be populated with "default", I suggest leave it as is. Meanwhile, to cover both http and https, should a new https-based test case be created, instead of revising the existing http test case?

Reply:
sure I will revert this change. @yangminzhu do we need to test https separately here ? We have done the unit testing separately though.

Reply:
It is using a namespace actually I think: https://github.com/istio/istio/blob/424695809a6c77e45642f5ec2090e66dd88b89b1/tests/integration/security/jwt_test.go#L324

Reply:
yeah, I think the namespace is used, and it makes sense to have a separate test.